lms/djangoapps/teams/static/teams/js/teams_features.js

```diff
@@ -101,6 +112,7 @@
                 var urlRocketChat = "/xblock/"+options.rocketChatLocator;
                 var urlApiCreateTeams = options.teamsCreateUrl;
                 var staff = options.userInfo.staff;
+                var teamsLocked = (options.teamsLocked == "true");
```

Comment:
Ok it's a string, here you validate if it's equal to the string "true". is there any way to send the value as Boolean from the template?

Reply:
there is no way that I knows